src/store/auth/reducer.tsx

```diff
@@ -35,6 +36,7 @@ export const authReducer = createReducer<AuthState, Action>(INIT_STATE)
         ...users,
         payload.user,
       ]);
+      draft.alertText = ''; // Maybe have an alert for "User Created"?
```

Comment:
Best to use TODOs:

// TODO(abradham): Consider adding an alert for "User Created"